src/doc/src/reference/manifest.md

```diff
@@ -530,6 +531,46 @@ both `src/bin/a.rs` and `src/bin/b.rs`:
 default-run = "a"
 ```
 
+#### The `lints` section
+
+Override the default level of lints from different tools by assigning them to a new level in a
+table, for example:
+```toml
+[lints.rust]
+unsafe = "forbid"
```

Comment:
I think `unsafe` is not a valid lint name, did you mean `unsafe_code`? Or is there an alias for a particular lint?